pkg/apply/testdata/apply/simple_ns/addons-repo.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: source.toolkit.fluxcd.io/v1alpha1
+kind: HelmRepository
+metadata:
+  name: microservice
+  namespace: simple
+spec:
+  interval: 1m0s
+  url: https://stefanprodan.github.io/podinfo
```

Comment:
I don't think this belongs here, and I'm reorganizing some of the testdata elements in a different PR.  Let's confer and find an appropriate home for this resource when we get a chance.

Reply:
let's discuss integration tests, Rajan had some comments on these